app/views/shared/_search_bar.html.erb

```diff
@@ -1,5 +1,5 @@
-<div id="user-search-bar" class="col-md-5 row pull-right">
```

Comment:
@henders, I've refactored it because I noticed that this partial could be reused everywhere, and not only for searching for users. 

But you're right about the `id`, I didn't remember we could end up with more than one search bar in the page. I removed the id, as it was only being used for styling purposes, and added a `search-bar`class to the class attribute to be able to style it from CSS.

Sounds good?